src/domain/usecases/interfaces/getMovieSuggestion.interface.ts

```
@@ -0,0 +1,12 @@
+import { MovieEntity } from "../../entities/";
+
+export interface GetMovieSuggestionUsecase {
+  exec: (
+    params: GetMovieSuggestionUsecase.Params,
+  ) => Promise<GetMovieSuggestionUsecase.Model>;
+}
+
+export namespace GetMovieSuggestionUsecase {
+  export type Params = String[];
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)